packages/schematics/src/collection/library/index.ts

```diff
@@ -304,6 +304,10 @@ describe('${options.moduleName}', () => {
           }
         };
       }),
+      updateJsonInTree(`${options.projectRoot}/ng-package.prod.json`, json => {
```

Comment:
Make sure the library is meant to be `publishable`. You can use a ternary to decide between this rule and a `noop()`

Also I think that the dist structure should match the directory structure.

```ts
return {
  ...json,
  dest: `${offsetFromRoot(options.projectRoot)}dist/${options.projectRoot}
};
```

Unit tests can test this :)